LittlePoops/little-poops-01.py

```diff
@@ -0,0 +1,35 @@
+# Write a function that takes a positive integer as an argument &
+# returns a Boolean indicating whether the argument is a power of three.
+
+# Hint: you might want to use recursion for this, which we haven't talked about yet. Exciting!
+
+def threepower(x,p):
+    p = int(p)
+    if x ** 3 == p:
+        return True
+    elif x >= p:
+        return False
+    elif x ** 3 < p:
+        x += 1
+        return threepower(x,p)
+
+
+def main():
+    trying = True
+    while trying:
+        p = input("Gimme a number:\n")
```

Comment:
It's best to separate your UI data from your program data. That is, try not to reuse the same input variable for both the number and the quit command. Among other things, that lets you immediately convert numeric inputs to the appropriate data type for validation.